### PR TITLE
adjust collector's ci test to send claim file to prod collector

### PR DESCRIPTION
**Pull request by shirmoran**

### CODE REVIEW

File: cnf-certification-test/tnf_config.yml [hunk 1]
```diff
@@ -36,6 +36,6 @@ servicesignorelist:
   - "new-pro-controller-manager-metrics-service"
   - "mysql"
 collectorAppEndPoint: ""
-executedBy: "default-executed-by"
+executedBy: ""
```
Comment (bnshr) [inline cnf-certification-test/tnf_config.yml:39]: Are these fields necessary to add with empty string?

**Review by bnshr — CHANGES_REQUESTED** [review]

File: cnf-certification-test/tnf_config.yml (hunk 1 above)
Comment (shirmoran) [reply cnf-certification-test/tnf_config.yml:39]: The partners are supposed to fill them.

Comment (bnshr) [reply cnf-certification-test/tnf_config.yml:39]: Yes, but this is the config for CI, so these fields can be empty. ExecutedBy is the mandatory, but the others not. So, we do not need those fields to be present in the config for CI test.

Comment (shirmoran) [reply cnf-certification-test/tnf_config.yml:39]: Though the tnf_config.yml file is not only for CI use, so won't it be more convenient for them to be there - so the partners just have to fill them in if they want.

Comment (shirmoran) [reply cnf-certification-test/tnf_config.yml:39]: and we should probably also add the ec2 endpoint as default for the `collectorAppEndPoint` in the tnf-config

Comment (bnshr) [reply cnf-certification-test/tnf_config.yml:39]: Partners dont need to fill ec2 endpoint. The code that parses this tnf_config should fill the collector_url and make request to there based on executedBy.

Comment (shirmoran) [reply cnf-certification-test/tnf_config.yml:39]: why based on executedBy? we run the CI testin rds too.. So I guess it's always ec2's endpoint. 
Though I agree it shouldn't be set in the tnf_config .

**Review by sebrandon1 — APPROVED** [review]

File: cnf-certification-test/tnf_config.yml (hunk 1 above)
Comment (bnshr) [reply cnf-certification-test/tnf_config.yml:39]: IIRC, this field was mandatory for claim upload, the rest is not.

Comment (bnshr) [reply cnf-certification-test/tnf_config.yml:39]: Ah, you are replacing these values. Looks good then!

**Review by bnshr — APPROVED** [review]

**Pull request merged**